app/models/manageiq/providers/vmware/infra_manager/refresh_worker/runner.rb

```diff
@@ -1,43 +1,35 @@
 class ManageIQ::Providers::Vmware::InfraManager::RefreshWorker::Runner < ManageIQ::Providers::BaseManager::RefreshWorker::Runner
-  # When using streaming_refresh we don't need to use the VimBrokerWorker
-  self.require_vim_broker           = !Settings.ems_refresh.vmwarews.streaming_refresh
-  self.delay_startup_for_vim_broker = !Settings.ems_refresh.vmwarews.streaming_refresh
+  self.require_vim_broker           = false
+  self.delay_startup_for_vim_broker = false
```

Comment:
If the default is false, can these just be removed?

Reply:
:+1: done